dev/ci/ci-basic-overlay.sh

```diff
@@ -214,11 +214,14 @@
 : "${equations_CI_ARCHIVEURL:=${equations_CI_GITURL}/archive}"
 
 ########################################################################
-# Elpi
+# Elpi + Hierarchy Builder
 ########################################################################
 : "${elpi_CI_REF:=coq-master}"
 : "${elpi_CI_GITURL:=https://github.com/LPCIC/coq-elpi}"
 : "${elpi_CI_ARCHIVEURL:=${elpi_CI_GITURL}/archive}"
+: "${elpi_hb_CI_REF:=coq-master}"
```

Comment:
Please leave a space
```suggestion

: "${elpi_hb_CI_REF:=coq-master}"
```